feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ec959a76336c8767f5cd4d.md

---
id: 64ec959a76336c8767f5cd4d
title: Step 20
challengeType: 0
dashedName: step-20
---

# --description--

> As you did in the last step, create another `p` element and interpolate the `date` you destructured as the text. Also, create a `strong` element inside the paragraph with the text `Date:`.

Reviewer: ```suggestion
Similarly to the previous step, create another `p` element, and interpolate the `date` you destructured as the text content. Inside this paragraph, create a `strong` element with the text `Date:`.
```
